files/ansible/tasks/set_ib_ip.yml

```diff
@@ -0,0 +1,87 @@
+---
+- name: Install IB support group
+  yum:
+        name: "@Infiniband Support"
+        state: present
+
+- name: Install IB support, test deps and parser deps
+  yum:
+        name: ['rsync', 'perftest', 'numactl', 'infiniband-diags', 'perftest', 'gperf', 'opensm']
+        state: present
+
+- name: Check if OHPC installed
+  yum:
+        list: ohpc-release
+  register: ohpc
+
+- block:
+    - name: Check if OHPC installed
+      yum:
+        list: mpitests-openmpi3
+      register: mpitests
```

Comment:
if it is installed, will this just drop? I think this check should be before checking for ohpc